server/controllers/user.controllers.js

```diff
@@ -121,9 +121,15 @@ const UserController = {
   },
 
   getAllRegisteredUsers(req, res, next) {
-    getAllRegisteredUsers()
+    const skip = req.params.count;
```

Comment:
This should be optional, so there should be a default value of 0 here. Also, can we make this offset instead of count so that it can be similar to the output defined.
```suggestion
    const skip = req.params.offset || 0;
```